doc/switcher.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "name": "latest",
+        "url": "https://param.holoviz.org/en/latest/"
+    },
+    {
+        "name": "Version 2.0.0 RC2",
+        "version": "v2.0.0rc2",
+        "url": "https://param.holoviz.org/en/2.0.0rc2/"
+    }
```

Comment:
Is this file manually written, or will it be auto-updated over time?

Reply:
Something will have to commit an update to the file. Will see what bokeh does.